Sources/CardPayments/APIRequests/CheckoutOrdersAPI.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+#if canImport(CorePayments)
+import CorePayments
+#endif
+
+/// This class coordinates networking logic for communicating with the v2/checkout/orders API.
+///
+/// Details on this PayPal API can be found in PPaaS under Merchant > Checkout > Orders > v2.
+class CheckoutOrdersAPI {
+    
+    // MARK: - Private Propertires
+    
+    private let coreConfig: CoreConfig
+    
+    // MARK: - Initializer
+    
+    init(coreConfig: CoreConfig) {
+        self.coreConfig = coreConfig
+    }
+    
+    // MARK: - Internal Methods
+        
+    func confirmPaymentSource(clientID: String, cardRequest: CardRequest) async throws -> ConfirmPaymentSourceResponse {
+        let apiClient = APIClient(coreConfig: coreConfig)
+        
+        let confirmData = ConfirmPaymentSourceRequest(cardRequest: cardRequest)
+        
+        let base64EncodedCredentials = Data(clientID.appending(":").utf8).base64EncodedString()
+        
+        // TODO: - Move JSON encoding into custom class, similar to HTTPResponseParser
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+        let body = try encoder.encode(confirmData)
+        
+        let restRequest = RESTRequest(
+            path: "/v2/checkout/orders/\(cardRequest.orderID)/confirm-payment-source",
+            method: .post,
+            headers: [
+                .contentType: "application/json", .acceptLanguage: "en_US",
+                .authorization: "Basic \(base64EncodedCredentials)"
```

Comment:
q: Is there a way we can make Auth header assignment more DRY?

Reply:
I think the best solution is allow API classes to make requests without having to think about auth–that'll make it easy to create new API classes in the future.

That is interesting about the tracking API though. On Android, for TrackingAPI we do still send the auth header. It isn't perfect since that header isn't required, but sending the header doesn't seem to cause any issues and it keeps the design clean. Most likely it will be ignored by the server.

Reply:
Addressed in commit 311ac819705d274487edbf873db3d6a0321ebce4